packages/lib/components/shared/note-screen-shared.ts

```diff
@@ -185,6 +186,7 @@ shared.noteComponent_change = function(comp: BaseNoteScreenComponent, propName:
 	newState.note = note;
 
 	comp.setState(newState);
+	comp.scheduleSave();
```

Comment:
It seems that the issue this fixes was already present: see https://github.com/laurent22/joplin/issues/10170